### DISCUSSION TITLE
Missing doc on how to install extensions in v10.10.4

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#21959] [Feature Requests]
### Summary

Used to v10.2.1 with tens of extensions, I migrate to v10.10.4 installed through CLI.

My test is on a panel extension:
1. There is a new admin/settings/extensions: what is it used for? In my case, it stays empty
2. Recompiling or creating a new extension with `npx create-directus-extension@latest` does not show any extension in the admin
3. auto reload does not work either despite `INFO: Watching extensions for changes...`
4. restarting directus leaves `directus_extensions` collection empty

What are the new requirements for extensions? Do they have to be registered some where?

### Motivation

to be able to upgrade to v10.10.4

### Detailed Design

just explain what to do

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

none

### Alternatives

none

### Adoption Strategy

immediate as soon it is clearly explained

**Comment by Pachat** [comment]
Finally found the answer [here](https://youtu.be/LBvNMJQ8cZU?feature=shared&t=97) 
So just had to move the directory to its parents.
Works great!

**Comment by br41nslug** [comment]
Beyond removing the long deprecated legacy folders (https://docs.directus.io/releases/breaking-changes.html#deprecated-typed-extension-folders) nothing changed in loading/installing extensions.

https://docs.directus.io/extensions/installing-extensions.html
